test/parallel/test-vm-context.js

```diff
@@ -50,16 +50,16 @@ assert.throws(() => {
 
 // Issue GH-1140:
 // Test runInContext signature
-let gh1140Exception;
+let gh1140Exp;
```

Comment:
I don't see the value in changing the variable name (no idea what `Exp` is supposed to mean).

Reply:
+1 ... generally we use more expressive variable names as much as possible.

Reply:
Proper fix is implemented in pull request #16116